--- build.d/targets/release.sh~	2018-12-18 06:34:11.000000000 +0100
+++ build.d/targets/release.sh	2020-09-23 01:11:10.000000000 +0200
@@ -85,7 +85,7 @@
     # Build distribution package
 
     build_target_invoke distribution build -s 10.5 -d 10.5 --installer-plugins-sdk=10.6 -c Release \
-                                           --kext=10.5 --kext=10.6 --kext="10.7->10.6" --kext="10.8->10.6" --kext=10.9 --kext="10.10->10.9" --kext=10.11 --kext="10.12->10.11" --kext="10.13->10.11" --kext="10.14->10.11" \
+                                           --kext=10.5 --kext=10.6 --kext="10.7->10.6" --kext="10.8->10.6" --kext=10.9 --kext="10.10->10.9" --kext=10.11 --kext="10.12->10.11" --kext="10.13->10.11" --kext="10.14->10.11" --kext="10.15->10.11" \
                                            --macfuse \
                                            --code-sign-identity="${BUILD_TARGET_OPTION_CODE_SIGN_IDENTITY}" \
                                            --product-sign-identity="${BUILD_TARGET_OPTION_PRODUCT_SIGN_IDENTITY}"
@@ -345,7 +345,7 @@
     <array>
 EOF
 
-    for macos_version in 10.5 10.6 10.7 10.8 10.9 10.10 10.11 10.12 10.13 10.14
+    for macos_version in 10.5 10.6 10.7 10.8 10.9 10.10 10.11 10.12 10.13 10.14 10.15
     do
 /bin/cat >> "${rules_plist_path}" <<EOF
         <dict>
